hack/components/bump-knmstate.sh

```diff
@@ -5,10 +5,14 @@ set -xeo pipefail
 source hack/components/yaml-utils.sh
 source hack/components/git-utils.sh
 
+echo kubernetes-nmstate
 NMSTATE_URL=$(yaml-utils::get_component_url nmstate)
 NMSTATE_COMMIT=$(yaml-utils::get_component_commit nmstate)
 NMSTATE_REPO=$(yaml-utils::get_component_repo ${NMSTATE_URL})
-NMSTATE_PATH=${GOPATH}/src/${NMSTATE_REPO}
+
+TEMP_DIR=$(git-utils::create_temp_path)
+trap "rm -rf ${TEMP_DIR}" EXIT
```

Comment:
i read somewhere that an error in a function wont trigger ERR, unless used set -E (instead of set -e),
well here its EXIT, maybe worth to make sure its indeed caught, even on other file if its the case and so on (there are arguments which control if set args are propagated between files and so on, so to see we are on road)

Reply:
I added a wrong cmd in `git-utils::get_component_tag` and the trap was indeed triggered

Reply:
Great thx

Reply:
thank you :)